Test/dafny0/formattingcheckfails.dfy.expect

```diff
@@ -0,0 +1 @@
+Should contain failing message
```

Comment:
Is this a placeholder for the real result?

Reply:
Good catch. This test wasn't run because I was using XFAIL. Now I'm using %exits-with and I can capture the error message. Fixed. Thanks !